contribute.md

```diff
@@ -0,0 +1,27 @@
+# How to contribute to the Copa Docker Desktop Extension
+
+Before proceeding, ensure that Docker Desktop is installed on your computer and using the WSL backend.
+
+## Building the copa-extenion image
+In order to run the extension locally, you need to build the image specified in the `/container` folder so that the frontend can call `docker run` on the image.
```

Comment:
```suggestion
In order to run the extension locally, you need to build the `copa-extension` image specified in the `/container` folder so that the frontend can call `docker run` on the image.
```